config/overlays/openshift/size-calculator/storage_size_calculator.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: storage-size-calculator
+spec:
+  selector:
+    matchLabels:
+      name: log-file-metric-exporter
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: log-file-metric-exporter
+    spec:
+      containers:
+      - command:
+        - /size-calculator
+        image: quay.io/sasagarw/storage-size-calculator:latest
```

Comment:
The defaul quay.io org should be openshift-logging: https://quay.io/repository/openshift-logging/storage-size-calculator